catalog/includes/OSC/Sites/Admin/site_conf.php

```diff
@@ -1,9 +1,9 @@
 <?php
 $ini = <<<EOD
-dir_root = ""
-http_server = ""
-http_path = ""
+dir_root = "/var/www/html/admin/"
+http_server = "http://13.57.225.143"
```

Comment:
No static IP

Reply:
i tried localhost. but that's not working. So i think user has to give their public ip here.

Reply:
If localhost not working then it should be a template, and value substituted with IP dynamically.